app/views/users/show.html.erb

```diff
@@ -1 +1,34 @@
-<h1>Here is the specified user</h1>
+<div>
+  <div class="user-container">
+    <%= image_tag(@user.photo + "images/rss.jpg", size: "50x50") %>
+
+    <div class="user-info">
+      <h5><%= @user.name %></h5>
+      <p>Number of posts : </p>
+    </div>
+
+  </div>
+  <div>
+    <div>
+      <h5>Bio</h5>
+      <p><%= @user.bio %></p>
+    </div>
+    <% @user.three_most_recent_posts.each do |post| %>
+      <div>
+        <%= link_to user_post_path(@user.id, post.id), class: "link-info" do %>
+          <h4><%= post.title %></h4>
+          <p><%= post.text %></p>
+          <div>
+            <small>Comments: <%= post.comments_counter || 0 %>,</small>
+            <small>Likes: <%= post.likes_counter || 0 %></small>
+          </div>
+        <% end %>
+      </div>
+    <% end %>
```

Comment:
- [ ] now when I try to navigate to the user's show page I got the following error please make sure to fix that in order to complete the requirements 
<img width="987" alt="Screenshot 2023-02-13 at 15 34 51" src="https://user-images.githubusercontent.com/61790587/218486839-575f901b-c44b-44b4-9369-e53e6bfcdb96.png">

Reply:
Hi @youmari !! 👋🏾 

Thanks for the quick review. I am confused because I don't have this issue locally. Everything is working fine. What can I do to solve it, please?

Reply:
@Samuellekpor you shouldn't do the concatenation in the view side the photo URL should be working without adding something to it right now you should handle the case where the `@user.photo is nil` that may return an error cause you are trying to contact a nil type with a string

Reply:
@Samuellekpor Nice adding the changes by the previous reviewer however when there is the bad link of a picture it returns this you should handle the case when for example the link of the image is broken or nil

Reply:
Alright, thanks @youmari. I will fix that as you said.